packages/page-pattern-modal/src/styles/page-pattern-modal.scss

```diff
@@ -35,12 +35,24 @@ $sidebar-width-desktop: 324px;
 
 // Overrides of the Gutenberg modal component
 .page-pattern-modal {
+	position: relative;
+
+	&.components-modal__frame {
+		max-height: none;
+	}
+
+	@media screen and ( max-width: $breakpoint-mobile ) {
+		height: calc( 100vh - 55px );
+		top: 50px; // don't get hidden by the adminbar
+	}
+
```

Comment:
Is this solving a different issue?

Reply:
It was a fix for small screens because I was seeing the adminbar sitting on top:

<img width="334" alt="Screen Shot 2021-06-16 at 14 22 25" src="https://user-images.githubusercontent.com/195089/122279981-4a3bda80-ceae-11eb-9bc2-e91815194b5c.png">

But that was Firefox, it sits on top properly in Chrome:

<img width="332" alt="Screen Shot 2021-06-16 at 14 23 13" src="https://user-images.githubusercontent.com/195089/122280055-63dd2200-ceae-11eb-85d8-fa9ebb4284ce.png">

Safari renders like Firefox. Neither allows overriding via z-index. So I think we should keep this, at least until we figure out a way to hide the adminbar when the modal is open on small screens.

Reply:
Can we connect this to the presence of the wpadminbar in some way? On the Simple Site I'm testing, the adminbar doesn't show, and this is what I see with this change applied

<img width="419" alt="image" src="https://user-images.githubusercontent.com/1699996/122287616-c0443f80-ceb6-11eb-9d40-0834b57847d8.png">